README.md

```diff
@@ -114,6 +119,22 @@ As of release 0.2.4 loader exclusions are no longer necessary to fix conflicts w
 
 If you do not see any coverage reports at the bottom of the page, even after clicking the enable coverage checkbox, check your console for errors. If you see something like `TypeError: blanket.instrumentSync is not a function` you need to update your version of blanket. Currently, we are pointing to a fork of blanket, but if you run `ember g ember-cli-blanket` it will update your app to point to the correct blanket version.
 
+### No Coverage Reported
```

Comment:
This is great, I'm thinking we should also add a TOC to the README to make these sections easier to find